source/features/swap-branches-on-compare.tsx

```diff
@@ -18,19 +17,34 @@ function init(): void {
 		references.unshift(select('.branch span')!.textContent!);
 	}
 
-	const icon = select('.range-editor .octicon-arrow-left')!;
-	icon.parentElement!.attributes['aria-label'].value += '.\nClick to swap.';
-	wrap(icon, <a href={buildRepoURL('compare/' + references.join('...'))} data-turbo-frame="repo-content-turbo-frame"/>);
+	const link = select('.js-toggle-range-editor-cross-repo')!;
+	link.after(
+		' or ',
+		<a href={buildRepoURL('compare/' + references.join('...'))}>
+			switch the base
+		</a>,
+	);
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isCompare,
 	],
 	exclude: [
+		// Disable on Two-dot Git diff comparison #4453
 		() => /\.\.+/.exec(location.pathname)?.[0]!.length === 2,
+		// Prefer the native "switch base" button if it is available
```

Comment:
Is there a native one? When does it appear? Maybe we should just reuse its style

Reply:
Oops, not button but link (it's the one in my first screenshot)

Reply:
Oh right